oxidation/libparsec/crates/types/src/lib.rs

```diff
@@ -3,6 +3,9 @@
 #[macro_use]
 extern crate lazy_static;
 
+#[cfg(any(test, feature = "types-fixtures"))]
```

Comment:
Would be good to add a comment to explain this recursive dependency issue that force us to define the fixtures here instead of in tests_fixture